.github/scripts/execute_single_benchmark.sh

```diff
@@ -15,13 +15,18 @@ echo "Config: ${CONFIG_NAME}, Dataset: ${DATASET_NAME}, Download Source: ${DATAS
 # Prepare the download URLs.
 if [ "$DATASET_NAME" == "skydio-8" ]; then
   export GDRIVE_FILEID='1mmM1p_NpL7-pnf3iHWeWVKpsm1pcBoD5'
+
 elif [ "$DATASET_NAME" == "skydio-32" ]; then
   export GDRIVE_FILEID='1BQ6jp0DD3D9yhTnrDoEddzlMYT0RRH68'
+
 elif [ "$DATASET_NAME" == "palace-fine-arts-281" ]; then
   WGET_URL1=http://vision.maths.lth.se/calledataset/fine_arts_palace/fine_arts_palace.zip
   WGET_URL2=http://vision.maths.lth.se/calledataset/fine_arts_palace/data.mat
   echo $WGET_URL1
   echo $WGET_URL2
+
+elif [ "$DATASET_NAME" == "2011205_rc3" ]; then
```

Comment:
not sure what that number "2011205" is, but if you really need it can we name it "rc3_2011205" and refer to this dataset as the rc3 dataset?

Reply:
"2011205" is the date (year/day_of_the_year) of the specific mission phase. I use the same names that NASA uses for each sequence. I'd like to keep with the NASA names if possible, but I do admit they're odd.

We can still refer to it as the rc3 dataset :)

Reply:
Thanks for the PR and for adding this Travis. Maybe you could add an inline bash comment explaining what RC3 stands for?

Reply:
Yes I can add a short description. Maybe we should add a short description for all of the datasets?

Reply:
that's a good idea

Reply:
Added description for rc3 and TODOs for the others